.codecov.yml

```diff
@@ -0,0 +1,6 @@
+coverage:
+  range: "80...100"
+  status:
+    project:
+      default:
+        threshold: 1
```

Comment:
I added this codecov.yml file to fix the issue of PRs failing if coverage drops by any amount. The threshold is set to 1%

Reply:
Thanks Ed